IssueTriage.md

```diff
@@ -0,0 +1,104 @@
+# Triage Guidance for WebNN Repo
+
+TODO: Consider removing these, or fit these into the documentation below.
+
+- **dependencies** - unused?
```

Comment:
I deleted this currently unused label.

The fewer the labels the more likely the group will use them (and use them correctly).